cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedInitialization.qll

```diff
@@ -993,7 +993,7 @@ class TranslatedConstructorBareInit extends TranslatedElement, TTranslatedConstr
 
   override TranslatedElement getChild(int id) { none() }
 
-  override Function getFunction() { result = getParent().getFunction() }
+  override Declaration getFunction() { result = getParent().getFunction() }
```

Comment:
This wasn't actually necessary to fix the inconsistency, but it was the only remaining `getFunction` implementation where:
- The return type was specified to be `Function`, and
- where result value wasn't _obviously_ a `Function`.